### PR TITLE
HL-252 | Show success notification upon saving the application

### PR DESCRIPTION
**Pull request by Nizar-Rahme**
## Description :sparkles:
- Show success notification upon saving the application

## Testing :alembic:
- Go to any step of the application, click on "Tallenna ja jatka myöhemmin" at the bottom of the page
- If the saving goes successfully, you will be redirected to the home page
- A success notification will be displayed with the applicant's name (if filled in) and the application number on it

## Screenshots :camera_flash:
image

## Additional notes :spiral_notepad:
- The application number is now not assigned due to a bug in the backend

### CODE REVIEW

File: frontend/benefit/applicant/src/hooks/useFormActions.tsx [hunk 1]
```diff
@@ -138,6 +138,23 @@ const useFormActions = (
         ? await updateApplication(data)
         : await createApplication(data);
 
+      const fullName = getFullName(
+        result?.employee?.first_name,
+        result?.employee?.last_name
+      );
+      const applicantName = fullName ? `(${fullName})` : 'ff';
```
Comment (nenonja) [inline frontend/benefit/applicant/src/hooks/useFormActions.tsx:145]: return of the function is never empty

File: frontend/benefit/applicant/src/utils/common.ts [hunk 2]
```diff
@@ -28,7 +28,7 @@ export const getApplicationStepFromString = (step: string): number => {
 export const getApplicationStepString = (step: number): string =>
   `step_${step}`;
 
-export const getApplicantFullName = (
+export const getFullName = (
   firstName: string | undefined,
   lastName: string | undefined
-): string => `${firstName || ''} ${lastName || ''}`;
+): string => [firstName, lastName].join(' ').trim();
```
[the remark this answers is not in the document]
Comment (Nizar-Rahme) [reply frontend/benefit/applicant/src/utils/common.ts:34]: According to [MDN documentation](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/join#description) (and to my experiments 😁 ), `null` and `undefined` values are converted to empty strings.

>Warning: If an element is undefined, null or an empty array [], it is converted to an empty string.

[the remark this answers is not in the document]
Comment (nenonja) [reply frontend/benefit/applicant/src/utils/common.ts:34]: Great, learning something new every day :)

**Review by nenonja — APPROVED** [review]
lgtm!

**Pull request merged**